src/modules/BootArch.rb

```diff
@@ -39,44 +41,29 @@ def DefaultKernelParams(resume)
         "globals",
         "additional_kernel_parameters"
       )
-      kernel_cmdline = Kernel.GetCmdLine
+      kernel_cmdline = Kernel.GetCmdLine.dup
 
       if Arch.i386 || Arch.x86_64
-        ret = kernel_cmdline != "" ? Ops.add(kernel_cmdline, " ") : ""
-        if resume != ""
-          ret = Ops.add(ret, Builtins.sformat("resume=%1 ", resume))
-        end
-        ret = Ops.add(Ops.add(ret, features), " ") if features != ""
-        if Builtins.regexpmatch(ret, "^(.* )?splash=[[:lower:]]+( .*)?$")
-          ret = Builtins.regexpsub(
-            ret,
-            "^((.* ))?splash=[[:lower:]]+(( .*)?)$",
-            "\\1 \\3"
-          )
-        end
-        ret = Ops.add(ret, "splash=silent quiet showopts")
```

Comment:
NP: If you already dupped Kernel.GetCmdLine, why use a different variable ```ret``` here instead of working with ```kernel_cmdline``` directly? It's not a big deal, anyways. Just curious.